.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,26 @@
+name: Continuous Integration
+on:
+  push:
+
+env:
+  RUBY_VERSION: '2.7.3'
```

Comment:
You can delete this. We set it in the design system because the sub-projects need something shared. But for this project `ruby/setup-ruby` should pick up the version in `.ruby-version` automatically.